src/content/getting started/observe-the-platform-metrics-and-logs/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Observe how your app is performing
+description: Check your app metrics and logs with the platform built-in observability tools.
```

Comment:
> What do we mean by "your app"? I guess GS managed apps and not really customer workloads (since they would have to use our observability tooling for that)?

Since the entire Getting Started is a journey, the app that you install in the step before is the one we use to show the logs or security feature in next sections